6.0-alpine/run.sh

```diff
@@ -6,8 +6,7 @@ if [ "${1:0:1}" != '-' ]; then
   exec "$@"
 fi
 
-exec java -jar lib/sonar-application-$SONAR_VERSION.jar \
-  ${SONARQUBE_JAVA_ARGS} \
+exec java ${SONARQUBE_JAVA_ARGS} -jar lib/sonar-application-$SONAR_VERSION.jar \
```

Comment:
`sonar-application-$SONAR_VERSION.jar` is just a bootstrapper, which will launch other java processes, and so I have doubts that setting `-Xmx` here would affect those child processes